app/hallebarde/revoke_expired_exchanges.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from datetime import datetime, timezone, timedelta
+
+from hallebarde import config
+from hallebarde.infrastructure import exchange_repository
+from hallebarde.usecases import revoke_exchange_command_handler
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handle(event: dict, context: dict) -> None:
+    maximum_creation_time = datetime.now(timezone.utc) - timedelta(days=config.EXPIRATION_DELAY_IN_DAYS)
+    logger.log(f'Revoking exchanges before {maximum_creation_time}')
```

Comment:
Why do you use logger.log ? this is `log` signature: `log(self, level, msg, *args, **kwargs)`

The first argument is an int to decide if you want to go INFO or DEBUG..

As is, you are not logging anything, I think